backend/bang-ggood/src/main/java/com/bang_ggood/checklist/domain/Checklist.java

```diff
@@ -0,0 +1,119 @@
+package com.bang_ggood.checklist.domain;
+
+import com.bang_ggood.BaseEntity;
+import com.bang_ggood.room.domain.Room;
+import com.bang_ggood.user.domain.User;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.Objects;
+
+@Table(name = "checklist")
+@Entity
+public class Checklist extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false)
+    private User user;
+
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false)
+    private Room room;
+
+    private Integer deposit;
+
+    private Integer rent;
+
+    private Integer contractTerm;
+
+    private String realEstate;
+
+    public Checklist(Integer deposit, Integer rent, Integer contractTerm, String realEstate) {
+        this.user = null;
+        this.room = null;
+        this.deposit = deposit;
+        this.rent = rent;
+        this.contractTerm = contractTerm;
+        this.realEstate = realEstate;
+    }
```

Comment:
기존 생성자를 활용해보는건 어떤가요?
```suggestion
    public Checklist(Integer deposit, Integer rent, Integer contractTerm, String realEstate) {
        this(null, null, deposit, rent, contractTerm, realEstate);
    }
```

Reply:
반영완료!